### ISSUE TITLE
Show SQL after results, not before?

### ISSUE THREAD

**Issue by hoehrmann** [AlexCovizzi/vscode-sqlite#67]
When you have long SQL text, e.g. due to elaborate comments in it, showing the results after the SQL in the `CTRL+Shift+Q` preview pane makes using it cumbersome. In fact, in a side-by-side view of the SQL and the pane, repeating the SQL with all formatting removed is not useful at all. There is a show/hide button, but it hides the results rather than the SQL and the setting is forgotten when hitting `CTRL+Shift+Q` again anyways.

**Comment by AlexCovizzi** [comment]
How about something like this:

![header_statement_collapsed](https://user-images.githubusercontent.com/16863733/57581005-32185680-74b1-11e9-9c47-da05738f4e9a.png)

and then you can click on the header to show the full statement
![header_statement_normal](https://user-images.githubusercontent.com/16863733/57581016-583df680-74b1-11e9-859f-79ca77353b4b.png)

**Comment by hoehrmann** [comment]
That would work for me if the state (collapsed/expanded) persists when I modify the SQL and press Ctrl+Shift+Q again!

**Comment by AlexCovizzi** [comment]
No, the state does not persist, I don't really have the time to implement it that way.

**Comment by hoehrmann** [comment]
Would a new setting be an option, so I could say "always start with the sql collapsed", "always start with the sql expanded"?

**Comment by AlexCovizzi** [comment]
I opted for leaving the statement collapsed by default, and expandable by clicking on it, without adding a new setting.
The reason is that I will probably add the persistence of the state collapsed/expanded in the future, so i don't want to add a setting that later will be removed.
